app/graphql/schema/queries/applications-by-sbi.js

```diff
@@ -0,0 +1,25 @@
+const cosmos = require('../../../cosmos')
+const { cosmosConfig } = require('../../../config')
+
+const applicationsBySbi = async (_root, args, context) => {
```

Comment:
Not suggesting you change this now as this is consistent, but maybe we should think about making these names verbs.  As the queries are increasing it's not clear which are gets and which will be updates.